modules/twinklespeedy.js

```diff
@@ -1497,6 +1497,11 @@ Twinkle.speedy.callbacks = {
 				editsummary = 'Requesting speedy deletion ([[WP:CSD#' + params.normalizeds[0].toUpperCase() + '|CSD ' + params.normalizeds[0].toUpperCase() + ']]).';
 			}
 
+			// Set the correct value for |ts= parameter in {{db-g13}}
+			if (params.normalizeds.indexOf('g13') === -1) {
```

Comment:
Shouldn't this be checking if if `!== -1`?

Reply:
Indeed! Thanks. No idea how my [test](https://en.wikipedia.org/w/index.php?title=Draft:Sandbox&oldid=929730959&diff=prev&diffmode=source) worked despite this mistake. Probably it was using the old morebits :( I really need to come up with a fool-proof way of ensuring the updated morebits has loaded before tests are run.